packages/tokens/scripts/generate-css-var-map.js

```diff
@@ -0,0 +1,119 @@
+const { argv } = require("node:process"); // For accessing command line arguments
+const { format, normalize } = require("node:path"); // For path manipulation
+const path = require("node:path"); // For path operations
+const { writeFile, mkdir } = require("node:fs/promises"); // For file system operations (async)
+
+// Constant for the prefix used in CSS variables
+const TELEGRAPH_VARIABLE_PREFIX = "tgph";
+
+/**
+ * Maps CSS variables to class names based on the provided object structure.
+ * @param {Object} obj - The object containing design tokens.
+ * @param {String} prefix - Prefix for the class name.
+ * @param {Boolean} noVariableNamePrefix - Flag to include/exclude the prefix in variable names.
+ * @returns {Object} - Mapped CSS variables to class names.
+ */
+const mapCssVarToClassName = (obj, prefix, noVariableNamePrefix) => {
+  const result = {};
+
+  // Helper function to recursively traverse the object
+  function traverseObject(obj, path = []) {
+    for (const key in obj) {
+      // Construct the path for the current key
+      const currentPath = path.concat(key);
+
+      if (
+        typeof obj[key] === "object" &&
+        obj[key] !== null &&
+        !Array.isArray(obj[key])
+      ) {
+        // Recurse into nested objects
+        traverseObject(obj[key], currentPath);
+      } else {
+        // Process leaf nodes
+        if (!currentPath.includes("dark")) {
+          const filteredPath = currentPath
+            .filter((t) => t !== "dark" && t !== "light" && t)
+            .join("-");
```

Comment:
Nit: I'd probably have this as `t && !["light", "dark"].includes(t)` and even consider extracting that array into a const